ui/src/components/Console.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import TerminalIcon from '../assets/terminal.svg';
+
+import styles from './Console.module.css';
+
+interface Props {
+  stdout: string;
+}
+
+const Console: React.FC<Props> = ({ stdout }) => {
+  const statements = stdout.trim().split('\n');
```

Comment:
We don't want to trim because empty print statements should still be reflected in the output!

Reply:
Ah good catch!

Reply:
Fixed in latest commit!